glib/hardcoded-paths.diff

```diff
@@ -11,11 +11,24 @@ index 2446d38..1042df2 100644
  
    /* Work out how many dirs we’re dealing with. */
    if (xdg_data_home != NULL || home != NULL)
+diff --git a/girepository/girepository.c b/girepository/girepository.c
+index a6a81716d..63bf2ec9e 100644
+--- a/girepository/girepository.c
++++ b/girepository/girepository.c
+@@ -300,6 +300,8 @@ gi_repository_init (GIRepository *repository)
+       typelib_dir = g_build_filename (libdir, "girepository-1.0", NULL);
+ 
+       g_ptr_array_add (repository->typelib_search_path, g_steal_pointer (&typelib_dir));
++
++      g_ptr_array_add (repository->typelib_search_path, g_strdup ("@@HOMEBREW_PREFIX@@/lib/girepository-1.0"));
+     }
+ 
+   repository->library_paths = g_ptr_array_new_null_terminated (1, g_free, TRUE);
 diff --git a/glib/gutils.c b/glib/gutils.c
```

Comment:
This seems fine to me, but can we drop the diff below this line? These changes don't seem needed.

Reply:
It's just tat I used a newer version of GLib (for the offsets).

I think we still want the change for `gutils.c`. It was also in the preceding patch.